### PR TITLE
feat : added header, sidebar , landing page and routes

### PR DESCRIPTION
**Pull request by SanjayJatti**
- Added landing page for notes app
- Added login and signup page screens
- Added header and sidebar components
- Added Routes 
langing-page-notes-app

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]

**Review by devananth — APPROVED** [review]

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link, NavLink } from "react-router-dom";
+import "./Header.css";
+
+const Header = () => {
+  return (
+    <div className="navbar-container">
+      <div className="nav-link header-title">
+        <Link to="/">
+          {" "}
```
Comment (parthgabani777) [inline src/Components/Header/Header.jsx:10]: You can remove comment

File: src/Pages/Authentication/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Header } from "../../Components/Header/Header.jsx"
+import { Link } from "react-router-dom";
+import "./Auth.css"
+
+export const Login = () => {
+  return (
+    <>
+    <Header/>
```
Comment (parthgabani777) [inline src/Pages/Authentication/Login.jsx]: rather than importing the Header in all pages, you can import in app file above routes

**Review by parthgabani777 — APPROVED** [review]

File: src/Pages/Authentication/Login.jsx (hunk 2 above)
Comment (SanjayJatti) [reply src/Pages/Authentication/Login.jsx]: Done 37f45c12b0cf181fe5599dfaa2dd4722907bf81e

**Comment by SanjayJatti** [comment]
Thanks for Reviews!

**Pull request merged**